certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "securityDefinitions": {
+    "basic": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basic": [],
+      "jwt": []
```

Comment:
@marcus-defazio Sorry for the late find - I have never seen this property, can you please let us know what is the use this property ?

Reply:
This was added when we generated our swagger file from our API code. I can remove it if you want.

Reply:
Do I need to make any changes @sriyen-msft ?

Reply:
> This was added when we generated our swagger file from our API code. I can remove it if you want.

Please remove it to make sure it would not affect anything